interface/weno/RxLog.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * weno rxlog.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+require_once("../globals.php");
+use OpenEMR\Core\Header;
+
+?>
+
+<html>
+<head>
```

Comment:
Below the <head> tag place the following line to give this script a title which will show on the Tab:
``````
<title><?php echo xlt("Rx Log"); ?></title>